src/main/resources/assets/carpet-addons-not-found/lang/en_us.json

```diff
@@ -18,6 +18,7 @@
   "carpet.rule.netheritePickaxeInstantMineEndStone.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine end stone type blocks.",
   "carpet.rule.netheritePickaxeInstantMineNetherBricks.desc": "A netherite pickaxe with efficiency V combined with the haste II status effect will instant mine nether brick type blocks.",
   "carpet.rule.phantomsObeyHostileMobCap.desc": "Phantoms will no longer spawn if the hostile mobcap is full. This is per player.",
+  "carpet.rule.portalSpectatorDelay.desc": "Amount of delay ticks to use a nether portal in spectator mode.",
```

Comment:
We should extend this description to include information about it only working with `spectatorPlayersUsePortals`. 

I’ve made the same comment for the README, so instead of an “additional note” in the README it should match this description exactly however you change it.